src/구명보트.js

```diff
@@ -8,11 +8,11 @@ function solution(people, limit) {
   while (lt < rt) {
```

Comment:
```suggestion
  while (lt <= rt) {
```

이렇게 조건을 수정하면 아래 21 ~ 23번째 코드를 삭제할 수 있습니다!

Reply:
이생각은 못했네요...